aat/src/aat/resources/features/common/acls/AccessControlList_RU.td.json

```diff
@@ -0,0 +1,9 @@
+{
+  "_guid_" : "AccessControlList_RU",
+  "_extends_": "AccessControlList__Base",
+
+  "create": false,
+  "read": true,
+  "update": true,
+  "delete": false
```

Comment:
Remove false values

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f